app/Mail/SchoolRegistrationConfirmation.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+use App\Models\School;
+use App\Models\User;
+
+class SchoolRegistrationConfirmation extends Mailable
```

Comment:
```suggestion
class SchoolRegistrationConfirmation extends Mailable implements ShouldQueue
```

Colocando o envio de email em filas